app/controllers/api/items_controller.rb

```diff
@@ -37,24 +37,20 @@ def post_to_realtime_extensions(items)
     extensions.each do |ext|
       content = ext.decoded_content
       next unless content
-      frequency = content['frequency']
-      if frequency == 'realtime'
-        post_to_extension(content['url'], items, ext)
-      end
+      post_to_extension(content['url'], items, ext, content['frequency'])
```

Comment:
We appear to be running this job now whether or not frequency == realtime. Seems like this check should be retained.

Reply:
We "force mute" it at the ExtensionJob invoke level so now it's moved to here: https://github.com/standardnotes/syncing-server/pull/106/files#diff-b2d6cad020de1805df8ddd050e8e159fR52

Reply:
Well it's not about muting, just about running. If I understand correctly, the previous code runs the job if frequency == realtime. The new code runs the job always, but emails user if frequency == realtime.

Reply:
previous code runs both realtime and backup:
- realtime in the line commented
- backup in here: https://github.com/standardnotes/syncing-server/pull/106/files#diff-b2d6cad020de1805df8ddd050e8e159fL68

or am I missing something?

Reply:
I'm not even sure the [`backup`](https://github.com/standardnotes/syncing-server/pull/106/files#diff-b2d6cad020de1805df8ddd050e8e159fR58) function is used tbh.

But this function in particular is `post_to_realtime_extensions`. So we want to loop through all extensions, and only execute the job if we determine it is a `realtime` extension. Without the if statement, we will execute this job even when it is a `daily` frequency extension. `realtime` extensions are what we designate things like note history as. This is because for every change, we want to backup these items in realtime. However, it seems like in this case, for every note change, we would run a `daily` backup job, which includes email backups and cloud backups.

Reply:
oh gotcha, thanks for the explanation - will apply a fix